lib/io/rdf/sparql/sparql-rdf-source.ts

```diff
@@ -35,7 +35,7 @@ export class RdfSparqlSource implements RdfSource {
     const query = createSparqlReverseProperty(iri, predicate);
     const quads = await fetchRdfQuadsBySparqlConstruct(
       this.httpFetch, this.endpoint, query);
-    return quads.map(quad => quad.subject);
+    return quads.map(quad => RdfObject.node(quad.subject));
```

Comment:
Subject should already be a node.

Reply:
You're right. I apologize again. Maybe it wasn't like that at some point in history because I'm sure I checked it.